translator/trace/jaeger/protospan_to_jaegerproto.go

```diff
@@ -138,18 +138,21 @@ func ocNodeAndResourceToJaegerProcessProto(node *commonpb.Node, resource *resour
 	if resource != nil {
 		resourceType := resource.GetType()
 		if resourceType != "" {
-			jTags = append(jTags, jaeger.KeyValue{
+			resourceTypeTag := jaeger.KeyValue{
 				Key:   opencensusResourceType,
 				VType: jaeger.ValueType_STRING,
 				VStr:  resourceType,
-			})
+			}
+			jTags = append(jTags, resourceTypeTag)
```

Comment:
I don't understand what is the difference other than declaring it as a separate variable? Does this fix a bug? I don't see it. Please clarify.